### PR TITLE
initial commit of ucs-service

### PR DESCRIPTION
**Pull request by geoff-reid**
@RackHD/corecommitters @tannoa2 @keedya @uppalk1

### CODE REVIEW

File: swagger/swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  /rackmount:
+    get:
+      tags:
+      - "default"
+      description: "Discover USC managed rackmount servers"
+      operationId: "controllers.default_controller.getRackmount"
+      parameters:
+      - name: "host"
+        in: "query"
+        description: "USC manger host name or IP"
```
Comment (rahmanmuhamad) [inline swagger/swagger.yaml]: look like a typo , you may want to change "manger" to "manager"

File: swagger/swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        required: false
+        type: "string"
+      - name: "user"
+        in: "query"
+        description: "UCS manger user name"
```
Comment (rahmanmuhamad) [inline swagger/swagger.yaml]: typo for "manger"

File: swagger/swagger.yaml (hunk 1 above)
Comment (rahmanmuhamad) [inline swagger/swagger.yaml]: look like a typo , you may want to change "manger" to "manager"

File: swagger/swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        required: false
+        type: "string"
+      - name: "password"
+        in: "query"
+        description: "UCS manger user password"
```
Comment (rahmanmuhamad) [inline swagger/swagger.yaml]: +1

**Comment by geoff-reid** [comment]
Fixed all the manger typos

**Pull request merged**